<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RDCPreferencesDialog</class>
 <widget class="QDialog" name="RDCPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>其他设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="bgDeleteCheckBox">
        <property name="text">
         <string>删除壁纸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="verifyCheckBox">
        <property name="text">
         <string>密码验证</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoConnectCheckBox">
        <property name="text">
         <string>自动同意连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>远程地址设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>远程服务器地址:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="serverAddrLineEdit">
        <property name="placeholderText">
         <string>服务器IP地址</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>远程服务器端口:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="serverPortLineEdit">
        <property name="placeholderText">
         <string>服务器端口号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>本地端口设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="screenPortLineEdit">
        <property name="placeholderText">
         <string>图像端口号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>图像端口:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>命令端口:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="commandPortLineEdit">
        <property name="placeholderText">
         <string>命令端口号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
